CHANGELOG.asciidoc

```diff
@@ -11,8 +11,8 @@ https://github.com/elastic/beats/compare/v7.6.2...v7.7.0[View commits]
 
 *Affecting all Beats*
 
-- Environment variables can no longer reference other environment variables or objects. {pull}15937{15937}
-- Change `aws_elb` autodiscovery provider field name from `elb_listener.*` to `aws.elb.*`. {issue}16219[16219] {pull}16402{16402}
+- Environment variables can no longer reference other environment variables or objects. {pull}15937[15937]
+- Change `aws_elb` autodiscovery provider field name from `elb_listener.*` to `aws.elb.*`. {issue}16219[16219] {pull}16402[16402]
 - Remove `AddDockerMetadata` and `AddKubernetesMetadata` processors from the `script` processor. They can still be used as normal processors in the configuration. {issue}16349[16349] {pull}16514[16514]
 
 *Journalbeat*
```

Comment:
@mikemadden42  The items under Journalbeat and Metricbeat here do not seem like breaking changes. Do I need ot move them to the correct sections under bug fixes?

Reply:
@urso Do these look like breaking changes to you?  Thanks.

Reply:
No, these belong into the 'Added' section.